### PR TITLE
Add background to error page

### PR DESCRIPTION
**Pull request by constanzauanini**
Desktop
Screen Shot 2022-08-22 at 16 48 32

Tablet
Screen Shot 2022-08-22 at 17 47 48

Mobile
Screen Shot 2022-08-22 at 17 47 56

### CODE REVIEW

**Comment by karthikbtf** [comment]
> Screen Shot 2022-08-22 at 15 51 06
> 
> I left the previous error. Should I change the error message in this page?

Design file has details https://www.figma.com/file/zfhbRTOi16UBS1Ro26iyya/Application-States?node-id=116%3A94
We can keep the error message but use the same styles and buttons like in the design file. See image below

Application States – Figma 2022-08-22 12-07-01

**Comment by karthikbtf** [comment]
> > Screen Shot 2022-08-22 at 15 51 06
> > I left the previous error. Should I change the error message in this page?
> 
> Design file has details https://www.figma.com/file/zfhbRTOi16UBS1Ro26iyya/Application-States?node-id=116%3A94 We can keep the error message but use the same styles and buttons like in the design file. See image below
> 
> Application States – Figma 2022-08-22 12-07-01

oops the contact-us button should go to the external site for support.. if they have trouble accessing our web-app contact-us page won't work.

**Comment by constanzauanini** [comment]
> > > Screen Shot 2022-08-22 at 15 51 06
> > > I left the previous error. Should I change the error message in this page?
> > 
> > 
> > Design file has details https://www.figma.com/file/zfhbRTOi16UBS1Ro26iyya/Application-States?node-id=116%3A94 We can keep the error message but use the same styles and buttons like in the design file. See image below
> > Application States – Figma 2022-08-22 12-07-01
> 
> oops the contact-us button should go to the external site for support.. if they have trouble accessing our web-app contact-us page won't work.

Updated screenshot

File: src/AppError.tsx [hunk 1]
```diff
@@ -1,25 +1,54 @@
...
     <div className={classes.main}>
       <Switch>
         <Route exact path={APP_PATHS.ERROR_FAILED_TO_FETCH_ORG_DATA}>
-          <ErrorBox title={strings.ORGANIZATION_DATA_NOT_AVAILABLE} text={strings.CONTACT_US_TO_RESOLVE_ISSUE} />
+          <Grid container justifyContent='center'>
+            <Box sx={{ margin: '0 auto', color: '#FFFFFF', width: '550px', textAlign: 'center' }}>
+              <Typography sx={{ fontSize: '40px', fontWeight: 600, paddingBottom: '32px' }}>
+                {strings.SOMETHING_WENT_WRONG_TITLE}
+              </Typography>
+              <Typography>{strings.ORGANIZATION_DATA_NOT_AVAILABLE}</Typography>
```
Comment (karthikbtf) [inline src/AppError.tsx:36]: Ugh.. ok I didn't know this was a hardcoded error. This seems wrong, might as well remove it.
If we were surfacing errors from the app .. that would be a different I guess.

**Review by karthikbtf — APPROVED** [review]
Have one comment. Looks good otherwise, thanks for doing this.
Can you copy @rachel-tf-design as well?
If you have screenshots for mobile/laptop that will help since design accounted for those form factors.

File: src/AppError.tsx (hunk 1 above)
Comment (constanzauanini) [reply src/AppError.tsx:36]: Yes. Agree. I will remove it

**Pull request merged**